datasets/private-schools-non-religious-cleaned.json

```diff
@@ -1,78 +1,1078 @@
-[
-    {
-        "School": "Abintra Montessori School",
-        "Address": "914 Davidson Dr. Nashville, TN 37205"
-    },
-    {
-        "School": "Battle Ground Academy",
-        "Address": "336 Earnest Rice Lane Franklin, TN 37069"
-    },
-    {
-        "School": "Benton Hall Academy",
-        "Address": "2422 Bethlehem Loop Road Franklin, TN 37069"
-    },
-    {
-        "School": "Casa dei Montessori",
-        "Address": "7646 Hwy. 70 S. Nashville, TN 37221"
-    },
-    {
-        "School": "Currey Ingram Academy",
-        "Address": "6544 Murray Lane Brentwood, TN 37027"
-    },
-    {
-        "School": "East Academy",
-        "Address": "108 Chapel Ave. Nashville, TN 37206"
-    },
-    {
-        "School": "Ensworth School",
-        "Address": "211 Ensworth Ave. Nashville, TN 37205"
-    },
-    {
-        "School": "Ensworth School",
-        "Address": "7401 Highway 100 Nashville, TN 37221"
-    },
-    {
-        "School": "Harding Academy",
-        "Address": "170 Windsor Dr. Nashville, TN 37205"
-    },
-    {
-        "School": "Harpeth Hall School",
-        "Address": "3801 Hobbs Rd. Nashville, TN 37215"
-    },
-    {
-        "School": "Linden Waldorf School",
-        "Address": "3201 Hillsboro Pike Nashville, TN 37215"
-    },
-    {
-        "School": "Montessori Academy",
-        "Address": "6021 Cloverland Dr. Brentwood, TN 37027"
-    },
-    {
-        "School": "Montessori East",
-        "Address": "801 Porter Road Nashville, TN 37206"
-    },
-    {
-        "School": "Montessori Centre",
-        "Address": "4608 Granny White Pike Nashville, TN 37220"
-    },
-    {
-        "School": "Montessori School of Franklin",
-        "Address": "1325 W. Main St., Ste. G Franklin, TN 37064"
-    },
-    {
-        "School": "Montgomery Bell Academy",
-        "Address": "4001 Harding Rd. Nashville, TN 37205"
-    },
-    {
-        "School": "Nashville Academy ",
-        "Address": "945 S. Douglas Ave. Nashville, TN 37204"
-    },
-    {
-        "School": "PB & J Day School",
-        "Address": "120 Werthan Circle Franklin, TN 37064"
-    },
-    {
-        "School": "University School of Nashville",
-        "Address": "2000 Edgehill Ave. Nashville, TN 37212"
-    }
-]
\ No newline at end of file
+
```

Comment:
Could you remove the HTML and add back the original datasets for this and `private-schools-religious-cleaned.json`?